index.html

```diff
@@ -1975,42 +2069,46 @@
 
     <section id="match_pattern"> <h3>Match patterns</h3>
       <p>
-        A match pattern is a string which can contain the
+        A <a>match pattern</a> is a string which can contain the
         <code>'*'</code> wildcard character.
```

Comment:
Nit: You can probably just say "A match pattern is defined by the following ABNF."